sdk/appservice/arm-appservice-rest/package.json

```diff
@@ -89,7 +89,7 @@
     "prettier": "2.2.1",
     "rimraf": "^3.0.0",
     "source-map-support": "^0.5.9",
-    "typedoc": "0.15.2",
+    "typedoc": "^0.22.17",
```

Comment:
@joheredi why does this package use typedoc?

Reply:
That's a valid question, and I was sort of wondering myself. It's being used in the `docs` npm script and nowhere else. Running that script generates docs, of course, but I have no idea why this particular package needs that.

Reply:
Individual package should stop having "typedoc" as their dev deps.  This one might be from code-gen a while ago.  There was also discussion on whether we should upgrade for 0.x.x versions.   PNPM consider them as available upgrades. However, 0.x.x versions are not as stable as non-zero major versions.

Reply:
> Individual package should stop having "typedoc" as their dev deps. This one might be from code-gen a while ago. There was also discussion on whether we should upgrade for 0.x.x versions. PNPM consider them as available upgrades. However, 0.x.x versions are not as stable as non-zero major versions.

SemVer has special handling for this, ^0.1.2 will be compatible with 0.1.3 but not 0.2.0. It looks for the first non zero number being incremented.

Reply:
OK, in that case, I am thinking we can drop this dependency altogether -- if that NPM script isn't getting used then it shouldn't be a problem.

Reply:
@joheredi @qiaozha is there any change needed in the code generator to not include "typedoc" in the dev dependency list?

Reply:
Yeah, we'll need a change in codegen. We'll need to remove the dependency and the `docs` script right?

Reply:
BTW this is an "experimental" management plane Rest Client

Reply:
Sent out this PR
https://github.com/Azure/autorest.typescript/pull/1419

Reply:
Thanks @joheredi, should I close this PR then and wait for things to come through from the codegen side?

Reply:
I have updated the codegen, in the meantime would you mind removing the dependency and script manually in this PR?

Reply:
Done!